electrum/commands.py

```diff
@@ -1475,6 +1475,8 @@ def get_parser():
     # daemon
     parser_daemon = subparsers.add_parser('daemon', help="Run Daemon")
     parser_daemon.add_argument("-d", "--detached", action="store_true", dest="detach", default=False, help="run daemon in detached mode")
+    parser_daemon.add_argument("--rpcsock", dest="rpcsocket", default='auto', help="what socket type to which to bind RPC daemon", choices=['unix', 'tcp', 'auto'])
+    parser_daemon.add_argument("--rpcsockpath", dest="rpcsocketpath", help="where to place RPC file socket")
```

Comment:
Let's make the config keys match the cli options.

```suggestion
    parser_daemon.add_argument("--rpcsock", dest="rpcsock", default='auto', help="what socket type to which to bind RPC daemon", choices=['unix', 'tcp', 'auto'])
    parser_daemon.add_argument("--rpcsockpath", dest="rpcsockpath", help="where to place RPC file socket")
```

Reply:
Disagree. For external options, being shorter is more important. For internal code, being descriptive is more important.

If you insist, I'm not strictly opposed to changing it though.

Reply:
> For external options, being shorter is more important. For internal code, being descriptive is more important.

Yes, I agree. However having the same name in both user-facing interfaces is another tradeoff (eases documentation, more intuitive, etc). I would argue the difference here is so small that it's worth it to unify the names.

I am fine with either the longer name or the shorter name, but I would prefer if they matched. We can use the longer names if you like that more.

Reply:
That is a fair point. No, in that case I prefer the shorter ones everywhere.